doc/source/whatsnew/v0.20.0.txt

```diff
@@ -189,6 +189,30 @@ You must enable this by setting the ``display.html.table_schema`` option to True
 .. _Table Schema: http://specs.frictionlessdata.io/json-table-schema/
 .. _nteract: http://nteract.io/
 
+.. _whatsnew_0200.enhancements.scipy_sparse:
+
+SciPy sparse matrix from/to SparseDataFrame
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Pandas now supports creating sparse dataframes directly from ``scipy.sparse.spmatrix`` instances (:issue:`4343`). See the :ref:`documentation <sparse.scipysparse>` for more information.
+
+All sparse formats are supported, but matrices that aren't in *COOrdinate* format will be converted to it, copying the data as needed.
```

Comment:
Linked to sparse module where coo_matrix is like the fifth word vertically and COOrdinate is about approximately hcentered on the screen.